### DISCUSSION TITLE
Seleniumbase not working with SOCKS5 proxies

### DISCUSSION THREAD

**Discussion by pd2871** [seleniumbase/SeleniumBase#2116] [Q&A] [answered]
@mdmintz 
I am trying to scrap using SOCK5S proxies as it works most of the time. I have been using undetected_chromedriver for using authenticated SOCK5S proxies and it works fine as of now. 
But now I want to do multithreading and use more proxies at the same time with different chrome profiles for each proxies. That seems to be not possible with uc as it needs to use pyautogui for filling the username and password and it seems not possible with threading.

So, I decided to use selenium-base but it is not working with SOCK5S proxies. 

Please help me on this.

![Screenshot (3010)](https://github.com/seleniumbase/SeleniumBase/assets/59336638/b1390a7c-af19-40d6-ab3f-f88780e388fd)

**Comment by mdmintz** [comment, marked as answer]
For socks4 or socks5 proxies, use these formats for the proxy_string:

Unauthenticated:

* ``socks4://IP_ADDRESS:PORT``

* ``socks5://IP_ADDRESS:PORT``

Authenticated:

* ``socks4://USER:PASS@IP_ADDRESS:PORT``

* ``socks5://USER:PASS@IP_ADDRESS:PORT``

**Comment by pd2871** [comment]
Hi, @mdmintz  
Thank you for the answer. In my case, the proxy was itself not working which was the reason for it to show like that. I changed the proxy and used the default format of seleniumbase `username:password@host:port` and it worked fine with SOCKS5.
